twenty-first/src/tip5/mod.rs

```diff
@@ -1067,6 +1089,50 @@ pub(crate) mod tests {
         assert_eq!(Tip5::offset_fermat_cube_map(255), 255);
     }
 
+    /// Ensure that the claims made in [`Tip5::mds_generated`] are true.
+    ///
+    /// In particular, `BFieldElement::Add` internally uses the equality
+    /// `a + b = a - (p - b)`. If `a` is a degenerate representation (i.e., the
+    /// Montgomery representation is larger than or equal to
+    /// [`BFieldElement::P`]), then a small enough `b` makes the result of the
```

Comment:
The parenthesis does not make sense to me. If the number if larger than or equal to $p$, then it is not in Montgomery representation.

Reply:
Maybe it becomes more clear when I remove the words “the Montgomery representation”?

“If `a` is a degenerate representation (i.e., is larger than or equal to [`BFieldElement::P`]), then […]”